sdk/python/tests/test_kfp_samples_report.txt

```diff
@@ -8,20 +8,20 @@ SUCCESS: default_value.py
 FAILURE: input_artifact_raw_value.py
 FAILURE: loop_over_lightweight_output.py
 SUCCESS: param_op_transform.py
-FAILURE: param_substitutions.py
+SUCCESS: param_substitutions.py
 SUCCESS: pipelineparams.py
 FAILURE: recursive_do_while.py
 SUCCESS: recursive_while.py
-FAILURE: resourceop_basic.py
+SUCCESS: resourceop_basic.py
 SUCCESS: sidecar.py
 SUCCESS: timeout.py
 SUCCESS: volume.py
-FAILURE: volume_snapshotop_rokurl.py
-FAILURE: volume_snapshotop_sequential.py
-FAILURE: volumeop_basic.py
-FAILURE: volumeop_dag.py
-FAILURE: volumeop_parallel.py
-FAILURE: volumeop_sequential.py
+SUCCESS: volume_snapshotop_rokurl.py
```

Comment:
We probably want to raise errors for volume_snapshotop and volumeop because we don't have any test for these Ops yet and the default volume manifest is generated with some Argo assumption.

Reply:
Addressed, all the volumeOp are failed now.